### PR TITLE
 Adding Help menu

### PR DESCRIPTION
**Pull request by chloegugli**

### CODE REVIEW

File: app/resources/config.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+"proactiveDashboardVersion": "7.36.0-SNAPSHOT",
```
Comment (jrochas) [inline app/resources/config.json]: How this version will be updated?

Comment (chloegugli) [reply app/resources/config.json]: I will also update the release project , build.gradle file

Comment (tobwiens) [reply app/resources/config.json]: The gradle version is updated automatically by the release process. 

But the config.json is not updated. Can you find a way to make this version being updated automatically or re-using the automatically updated properties. I believe @paraita worked on something similar at some point.

File: app/scripts/main.js [hunk 2]
```diff
@@ -115,10 +115,26 @@ mainCtrl.factory('MainService', function ($http, $interval, $rootScope, $state)
...
+
+    $scope.displayAbout = function(){
+        $http.get('resources/config.json')
+            .success(function (response) {
+                $scope.dashboardVersion = response.proactiveDashboardVersion;
+            })
+            .error(function (response) {
+                $scope.dashboardVersion = "not available";
+            });
+        var url = window.location.href;
```
Comment (tobwiens) [inline app/scripts/main.js]: Can you put this logic into a separate logic or a helper class/service. 

Usually when I am in the automation dashboard I have a url like this `https://trydev.activeeon.com/automation-dashboard/#/portal/workflow-automation`. Not sure if this code works reliably under all circumstances. 

Maybe something like:
```
var windowLocation = window.location;
var protocol = windowLocation.protocol
var host = windowLocation.host
// and construct from here
```

Comment (chloegugli) [reply app/scripts/main.js]: -Config.json will be updated once this PR is merged : https://bitbucket.org/activeeon/release/pull-requests/84/updating-version-of-proactive-dashboard-in/diff

File: app/resources/config.json (hunk 1 above)
Comment (tobwiens) [reply app/resources/config.json]: If it is not an easy fix we can fix this in another PR.

**Pull request merged**